endorS.py

```diff
@@ -11,111 +11,248 @@
     #Extract number of mapped reads pre-quality filtering:
-    mappedPre = float((re.findall(r'([0-9]+) \+ [0-9]+ mapped ',contentsPre))[0])
-    #Calculation of endogenous DNA pre-quality filtering:
+    mappedRaw = float((re.findall(r'([0-9]+) \+ [0-9]+ mapped ',contentsRaw))[0])
+
+    #Calculate Percentage on target raw (aka endogenous raw)
     if totalReads == 0.0:
-        endogenousPre = 0.000000
-        print("WARNING: no reads in the fastq input, Endogenous DNA raw (%) set to 0.000000")
-    elif mappedPre == 0.0:
-        endogenousPre = 0.000000
-        print("WARNING: no mapped reads, Endogenous DNA raw (%) set to 0.000000")
+        endogenousRaw = 0.000000
+        print("WARNING: no reads in the fastq input, Percent on target raw (%) set to 0.000000")
+    elif mappedRaw == 0.0:
+        endogenousRaw = 0.000000
+        print("WARNING: no mapped reads, Percent on target raw (%) set to 0.000000")
     else:
-        endogenousPre = float("{0:.6f}".format(round((mappedPre / totalReads * 100), 6)))
+        endogenousRaw = float("{0:.6f}".format(round((mappedRaw / totalReads * 100), 6)))
 except:
-    print("Incorrect input, please provide at least a samtools flag stats as input\nRun:\npython endorS.py --help \nfor more information on how to run this script")
-    sys.exit()
-#Check if the samtools stats post-quality filtering have been provided:
+    if args.verbose:
+        print("No samtools flagstat --raw provided. \nWARNING: none of the Percent on target stats will be calculated")
+
 try:
-    #Open the samtools flag stats post-quality filtering:
-    with open(args.samtoolsfiles[1], 'r') as post:
-        contentsPost = post.read()
+    with open(args.qualityfiltered, 'r') as qF:
+        contentsqF = qF.read()
     #Extract number of mapped reads post-quality filtering:
-    mappedPost = float((re.findall(r'([0-9]+) \+ [0-9]+ mapped',contentsPost))[0])
-    #Calculation of endogenous DNA post-quality filtering:
-    if totalReads == 0.0:
-        endogenousPost = 0.000000
-        print("WARNING: no reads in the fastq input, Endogenous DNA raw (%) set to 0.000000")
-    elif mappedPost == 0.0:
-        endogenousPost = 0.000000
-        print("WARNING: no mapped reads, Endogenous DNA modified (%) set to 0.000000")
-    else:
-        endogenousPost = float("{0:.6f}".format(round((mappedPost / totalReads * 100),6)))
+    mappedPost = float((re.findall(r'([0-9]+) \+ [0-9]+ mapped',contentsqF))[0])
+    #Calculation of Percent on target quality filtered (aka endogenous DNA post-quality filtering):
+    if args.raw is not None:
+        if totalReads == 0.0:
+            endogenousQF = 0.000000
+            print("WARNING: no reads in the fastq input, Percent on target raw (%) set to 0.000000")
+        elif mappedPost == 0.0:
+            endogenousQF = 0.000000
+            print("WARNING: no mapped reads, Percent on target modified (%) set to 0.000000")
```

Comment:
shouldn't modified be `quality filtered` here for consistency?

also

```suggestion
            print("WARNING: no mapped reads, percent on target modified (%) set to 0.000000")
```